packages/components/src/SquigglePlayground.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FC, useState } from "react";
+import ReactDOM from "react-dom";
+import { SquiggleChart } from "./SquiggleChart";
+import CodeEditor from "./CodeEditor";
+import { Form, Input, Card, Row, Col } from "antd";
+import "antd/dist/antd.css";
+
+interface FieldFloatProps {
+  label: string;
+  className?: string;
+  value: number;
+  onChange: (value: number) => void;
+}
+
+function FieldFloat(Props: FieldFloatProps) {
+  let [contents, setContents] = useState(Props.value + "");
+  return (
+    <Form.Item label={Props.label}>
+      <Input
+        value={contents}
+        className={Props.className ? Props.className : ""}
+        onChange={(e) => setContents(e.target.value)}
+        onBlur={(_) => {
+          let result = parseFloat(contents);
+          if (result != NaN) {
```

Comment:
I suggest using Lodash and then using _.isFinite or similar. The Lodash _.isX functions will catch more than just NaN. (For example, if the result is negative, infinity, false, etc.)
Generally, I'm paranoid about catching JS values manually. (Like this)